src/main/java/com/homeaway/devtools/jenkins/testing/WholeClasspathPipelineExtensionDetector.java

```diff
@@ -77,10 +81,10 @@
 				}
 			} else {
 				LOG.warn(
-					"Failed to get some classes of type [{}] in package [{}]. For detailed error messages, set the system property PipelineExtensionDetector.expandFailures=true. Failures: [{}]",
-					_supertype,
-					_package,
-					failures.keySet() );
+						"Failed to get some classes of type [{}] in package [{}]. For detailed error messages, set the system property PipelineExtensionDetector.expandFailures=true. Failures: [{}]",
+						_supertype,
+						_package,
+						failures.keySet() );
```

Comment:
Extra layer of indentation added here.

```suggestion
					"Failed to get some classes of type [{}] in package [{}]. For detailed error messages, set the system property PipelineExtensionDetector.expandFailures=true. Failures: [{}]",
					_supertype,
					_package,
					failures.keySet() );
```